setupini.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+
+###This shell script is used to setup initial environment during login. This checks whether a psql database is up or not. Or,
+### installed or not. If not, it will start the database or create a new one
+### Checks recent python version. If not upgraded, it upgrades it and set it as default alternative
+### Install required dependencies from requirements.txt
+
+### Database installation check
+
+db_exist=`which psql | wc -l`
```

Comment:
Piping to `wc -l` is unnecessary. `which` exits with exit code 0 if `psql` was found and 1 if not. So you could just do:
```
if which psql; then
    # postgresql is installed, do something
fi
```

You could also completely avoid using `which` (which spawns a new process just to check for a file in `$PATH`) by using shell built-in, but I don't think that matters much here.